src/dom.ts

```diff
@@ -1,13 +1,14 @@
+// tslint:disable
 /**
  * Utility functions to make DOM manipulation easier.
  */
 
 const NODE_TYPE = {
   ELEMENT_NODE: 1,
-  TEXT_NODE: 3
+  TEXT_NODE: 3,
 };
 
-export default function dom(el) {
+export default function dom(el: any) {
```

Comment:
Is `: any` used here so you do not have to `import "dom"`?

Reply:
no there were just a lot of functions in here so i didn't want to make a definition file, and after trying for a few minutes to legitimately add types to all this i gave up and switched to any. a lot of this is only manually tested and i'm scared to change the code too much to make typescript happy